x-pack/test/saved_object_api_integration/common/suites/resolve_import_errors.ts

```diff
@@ -61,34 +62,75 @@ export const TEST_CASES: Record<string, ResolveImportErrorsTestCase> = Object.fr
     type: 'sharedtype',
     id: `conflict_1a`,
     originId: `conflict_1`,
+    destinationId: 'some-random-id',
     expectedNewId: 'some-random-id',
   }),
   CONFLICT_1B_OBJ: Object.freeze({
     type: 'sharedtype',
     id: `conflict_1b`,
     originId: `conflict_1`,
+    destinationId: 'another-random-id',
     expectedNewId: 'another-random-id',
   }),
   CONFLICT_2C_OBJ: Object.freeze({
     type: 'sharedtype',
     id: `conflict_2c`,
     originId: `conflict_2`,
+    destinationId: `conflict_2a`,
     expectedNewId: `conflict_2a`,
   }),
+  CONFLICT_2D_OBJ: Object.freeze({
+    type: 'sharedtype',
+    id: `conflict_2d`,
+    originId: `conflict_2`,
+    // destinationId is undefined on purpose
+    expectedNewId: `conflict_2b`, // since conflict_2c was matched with conflict_2a, this (conflict_2d will result in a regular inexact match conflict with conflict_2b
```

Comment:
__nit:__?
```suggestion
    expectedNewId: `conflict_2b`, // since conflict_2c was matched with conflict_2a, this (conflict_2d) will result in a regular inexact match conflict with conflict_2b
```